climada/entity/exposures/crop_production.py

```diff
@@ -227,7 +259,7 @@ def set_from_isimip_netcdf(self, input_dir=None, filename=None, hist_mean=None,
 
         # The indeces of the yearrange to be extracted are determined
         time_idx = (int(yearrange[0] - yearchunk['startyear']),
-                             int(yearrange[1] - yearchunk['startyear']))
+                    int(yearrange[1] - yearchunk['startyear']))
 
         # The area covered by a grid cell is calculated depending on the latitude
```

Comment:
That's a typical util-function case, since calculating the area from degrees spanned could also be useful for others / more general operation.
Could you refactor that and put it in util.coordinates?
Sth in the line of `util.coordinates.deg_from_dist()` and `util.coordinates.dist_approx()`

Reply:
> I guess the whole module per se has been reviewed already, right? And this is just about the newest changes? Because from the code and length of both the module itself and the length of the functions, this would take me much longer to actually understand and review in-depth

Hi Evelyn, yes there has been a more thorough review of the module before. While there are probably still code architectural aspects that could be optimized, this PR is really about the fixes and small updates in the commits above. Thanks for taking the time to review :)